schema.sql

```
@@ -171,3 +171,16 @@ CREATE TABLE IF NOT EXISTS message_requests (
+
+-- Chat messages table (virtual classroom chat persistence)
+CREATE TABLE IF NOT EXISTS chat_message (
+    id           TEXT PRIMARY KEY,
+    classroom_id TEXT NOT NULL,
+    user_id      TEXT NOT NULL,
+    display_name TEXT,
+    content      TEXT NOT NULL,
+    created_at   TEXT NOT NULL DEFAULT (datetime('now'))
+);
+
+CREATE INDEX IF NOT EXISTS idx_chat_classroom ON chat_message(classroom_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)